lib.ftd

```diff
@@ -1946,74 +2230,190 @@ padding-horizontal.px: 12
-anchor: parent
-top.percent: 24
-z-index: -1
 width.fixed.px: 300
-align-content: center
-align-self: center
-
--- ftd.image:
-width: fill-container
-src: $assets.files.static.lezgoo.how-it-works.svg
+background.image: $bg-image
+height.fixed.px: 300
 align-self: center
 
--- end: ftd.column
-
--- ftd.column:
-width.fixed.percent: 85
+-- ftd.row:
 align-content: center
-min-width.fixed.px: 300
+width: fill-container
+height.fixed.px: 120
 
--- ftd.text: $works-cell.top-cell
+-- ftd.text: $works-cell-desktop.top-cell
 role: $inherited.types.blockquote
-color: $inherited.colors.text-strong
+color: $inherited.colors.custom.one
 padding-vertical.px: 13
 padding-horizontal.px: 17
-width.fixed.percent: 100
-margin-left.percent: 24
 
--- ftd.text: $works-cell.bottom-cell
+
+-- end: ftd.row
+
+-- ftd.row:
+align-content: center
+width: fill-container
+height.fixed.px: 120
+
+-- ftd.text: $works-cell-desktop.bottom-cell
 role: $inherited.types.blockquote
 color: $inherited.colors.background.step-1
 padding-vertical.px: 13
 padding-horizontal.px: 17
-width.fixed.percent: 100
+
+-- end: ftd.row
 
 -- end: ftd.column
 
 -- end: ftd.column
 
 -- ftd.image:
-src: $works-cell.right-src
-padding-horizontal.px: 12
+src: $works-cell-desktop.right-src
+padding-right.px: 12
 align-self: center
+width: fill-container
+margin-right.px: -44
 
 -- end: ftd.row
 
--- end: works-cell
+-- end: works-cell-desktop
+
+
+
+
+
+
+
+
+
+
+-- component works-cell-mobile:
+caption title:
+integer number:
+ftd.image-src left-src:
+ftd.image-src right-src:
+optional ftd.image-src left-mob-src:
+optional ftd.image-src right-mob-src:
+string top-cell:
+string bottom-cell:
+optional integer index:
+optional integer reset:
+
+-- ftd.column:
+width: fill-container
+margin-top.px: 25
+margin-bottom.px: 24
+
+-- ftd.row:
+width: fill-container
+spacing: space-between
+
+-- ftd.image:
+if: { works-cell-mobile.left-mob-src == NULL }
+src: $works-cell-mobile.left-src
+width.fixed.percent: 27
+height: auto
 
+-- ftd.image:
+if: { works-cell-mobile.left-mob-src != NULL }
+src: $works-cell-mobile.left-mob-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.right-mob-src == NULL }
+src: $works-cell-mobile.right-src
+width.fixed.percent: 27
+height: auto
+
+-- ftd.image:
+if: { works-cell-mobile.right-mob-src != NULL }
+src: $works-cell-mobile.right-mob-src
+width.fixed.percent: 27
+height: auto
+
+-- end: ftd.row
+
+-- ftd.row:
+background.solid: $inherited.colors.border
+padding-vertical.px: 7
+padding-horizontal.px: 12
+width: fill-container
+spacing.fixed.px: 6
+
+-- ftd.row:
+spacing.fixed.px: 2
+
+
+-- ftd.integer: $works-cell-mobile.number
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+
+-- ftd.text: .
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+
+-- end: ftd.row
+
+-- ftd.text: $works-cell-mobile.title
+role: $fonts.size-19
+color: $inherited.colors.background.step-1
+width: fill-container
+
+-- end: ftd.row
```

Comment:
Given attribute which applies to child like `color` and `role` etc, should be given to parent only. Not to individual child. 

```suggestion
-- ftd.row:
background.solid: $inherited.colors.border
padding-vertical.px: 7
padding-horizontal.px: 12
width: fill-container
spacing.fixed.px: 6
role: $fonts.size-19
color: $inherited.colors.background.step-1

-- ftd.row:
spacing.fixed.px: 2

-- ftd.integer: $works-cell-mobile.number

-- ftd.text: .

-- end: ftd.row

-- ftd.text: $works-cell-mobile.title
width: fill-container

-- end: ftd.row
```